mailreply00.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 Matthew Holt
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package acmez
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mholt/acmez/acme"
+)
+
+// MailChallangeReplyGen builds email body with headers to reply MailReply00
+// challange Response email. This fucntion just build body, and sendung
+// message have to done by caller of this function
+// mailSubject and messageId is such from challange mail,
+// and if there is no reply-to header in challange email, replyto parametere should be ""
+
```

Comment:
There shouldn't be a newline between a godoc comment and the function attached to it

```suggestion
```